pkgs/by-name/fo/fortune-kind/package.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, libiconv
+, makeWrapper
+, installShellFiles
+, fortuneAlias ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fortune-kind";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "cafkafk";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-q4r1Qdyh2L1vTi+CKln+a9lKGgyRAhvg1aJRd0CIEJs=";
+  };
+
+  cargoHash = "sha256-ArVOGWaKbge8nxxGpAT8TttFZYJpuFF5u/tHYC9Qkeo=";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
```

```suggestion
  nativeBuildInputs = [ makeBinaryWrapper installShellFiles ];
```
slightly preferred if possible, as mentioned in https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper

---

OHHH didn't know this, that's cool!